src/index.js

```diff
@@ -122,6 +123,16 @@ class PeerId {
     return this._idB58String
```

Comment:
This will now become rarely used. Should we generate on demand instead of in the constructor?

Reply:
It is used in `isPeerId` and changing this triggers a bigger refactor.
I'd like to keep it as-is in this PR and change it in `Stage 2` of https://github.com/libp2p/specs/issues/216
(when we "Format peer IDs as CIDs by default.")